docs/platform/visual/analyzer-viz-api.md

```diff
@@ -23,8 +23,8 @@ while still side-by-side with the _previous_ Visualization API version.
 The Analyzer application supports visualizations of both formats,
 allowing you to convert any custom visualizations of the previous format at your own pace.
 
-All [stock visualizations](https://help.pentaho.com/Documentation/8.3/Products/Visualizations_for_Analyzer), 
-with the exception of the [Geo Map](https://help.pentaho.com/Documentation/8.3/Products/Visualizations_for_Analyzer#Geo_map_visualization), 
+All [stock visualizations](https://help.pentaho.com/Documentation/9.4/Products/Visualizations_for_Analyzer), 
+with the exception of the [Geo Map](https://help.pentaho.com/Documentation/9.4/Products/Visualizations_for_Analyzer#Geo_map_visualization), 
```

Comment:
The correct base URL is https://help.hitachivantara.com/Documentation/Pentaho/9.4/Products/Visualizations_for_Analyzer

Reply:
https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1597